pkg/oauth/registry/oauthaccesstoken/strategy.go

```diff
@@ -57,6 +62,11 @@ func (s strategy) Validate(ctx kapi.Context, obj runtime.Object) field.ErrorList
 	return validationErrors
 }
 
+// ValidateUpdate validates an update
+func (s strategy) ValidateUpdate(ctx kapi.Context, obj, old runtime.Object) field.ErrorList {
+	return kvalidation.ValidateImmutableField(obj, old, &field.Path{})
```

Comment:
Actually want a validate update method that checks the fields we want immutable. Doing it this way means things like uid, creationTimestamp, etc, aren't defaulted from the existing object, which is inconsistent with all other objects